### PR TITLE
More built in interactors

### PR DESCRIPTION
**Pull request by jnicklas**
See #379

This adds two important interactors, `Button` and `TextField`. It also adds the `fillIn` function which can be used as an action to fill in a field, and can be re-used for the various types of field.

### CODE REVIEW

File: packages/interactor/src/fill-in.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+export function fillIn(element: HTMLInputElement, value: string) {
+  let Event = element.ownerDocument.defaultView?.Event || window.Event;
+
+  element.focus();
+
+  clearText(element);
+  enterText(element, value);
+
+  element.dispatchEvent(new Event('change', { bubbles: true, cancelable: false }));
```
Comment (cowboyd) [inline packages/interactor/src/fill-in.ts]: IIRC, the change event is only fired if the value is different from what it was before?

**Review by cowboyd — COMMENTED** [review]
More interactors == more fun 😅

File: packages/interactor/src/fill-in.ts (hunk 1 above)
Comment (jnicklas) [reply packages/interactor/src/fill-in.ts]: Nice catch!

**Review by cowboyd — APPROVED** [review]
Looks good. I've noticed that the `cli` tests seem to be timing out with some regularity. Not exactly sure what's going on there, but it's something we need to look into.

**Pull request merged**